put_message_on_bucket.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+import json
+
+from google.cloud import pubsub_v1, storage
+from termcolor import colored
+
+
+def main(subscription_name, project_id, number_messages, id_search, action, bucket_name):
+    subscriber = pubsub_v1.SubscriberClient()
+    response = []
+
+    subscription_path = subscriber.subscription_path(
+        project_id, subscription_name)
+    for _ in range(number_messages):
+        message = subscriber.pull(subscription_path, max_messages=1, return_immediately=True)
+        if not message.received_messages:
+            break
+        response.extend(message.received_messages)
+    if not response:
+        print(colored('No messages on pubsub ', 'red'))
+        return
+    elif id_search:
+        matching_id_messages = [msg for msg in response if id_search == msg.message.message_id]
+        if matching_id_messages:
+            if action == 'MOVE':
+                client = storage.Client()
+                bucket = client.get_bucket(bucket_name)
+                pubsub_dict = {'message_id': matching_id_messages[0].message.message_id,
+                               'data': matching_id_messages[0].message.data.decode()}
```

Comment:
Could use `'data': json.loads(matching_id_messages[0].message.data)}` here instead to prevent escaping the double-quotes when it gets uploaded to the bucket?

Reply:
I'll have a go at doing it that way. Think there was a reason I decoded it